threeML/plugins/OGIP/eventlist.py

```diff
@@ -461,41 +453,35 @@ def peek(self):
             info_dict['Active Polynomial Counts'] = self._poly_counts.sum()
             info_dict['Unbinned Fit'] = self._unbinned
 
-            sig =Significance(self._counts.sum(), self._poly_counts.sum())
+            sig = Significance(self._counts.sum(), self._poly_counts.sum())
 
-            bkg_sig = np.sqrt( (self._poly_count_err**2).sum())
+            bkg_sig = np.sqrt((self._poly_count_err ** 2).sum())
 
-
-            info_dict['Li and Ma Sigma'] = sig.li_and_ma_equivalent_for_gaussian_background(bkg_sig)  # not sure if li and ma applies here
+            # too provocative?
+            info_dict['Vianello Significance'] = sig.li_and_ma_equivalent_for_gaussian_background(bkg_sig)
```

Comment:
yes, too provocative! :-)

Let me change it simply to "Significance" for the moment.

I'll accept this pull request, and change it as a new commit.